client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.js

```diff
@@ -515,3 +522,38 @@ function GSuiteDiscountCallout( { item } ) {
 	}
 	return null;
 }
+function DiscountForFirstYearOnly( { item } ) {
+	const translate = useTranslate();
+	const hasDiscount = !! item.wpcom_meta.item_original_cost_integer;
```

Comment:
This can be a bit more readable with (coercion happens here automatically to compare with a string "0"
```javscript
const hasDiscount = item.wpcom_meta.item_original_cost_integer != 0; 
```